addons/mail/models/mail_thread.py

```diff
@@ -2685,15 +2709,17 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, mo
             'is_html_empty': is_html_empty,
         }
 
-    def _notify_by_email_get_base_mail_values(self, message, additional_values=None):
+    def _notify_by_email_get_base_mail_values(self, message, additional_values=None, model_description=None):
         """ Add model-specific values to the dictionary used to create the
         notification email. Its base behavior is to compute model-specific
         headers.
 
         :param dict base_mail_values: base mail.mail values, holding message
         to notify (mail_message_id and its fields), server, references, subject.
         """
-        mail_subject = message.subject or (message.record_name and 'Re: %s' % message.record_name) # in cache, no queries
+        mail_subject = message.subject or self._message_compute_subject(record_names={message.res_id: message.record_name},
```

Comment:
Would use a if-else statement (my linter always yells about doing "or" on strings, he formatted me), but that's a personal taste I think ^^ .

Reply:
@tde-banana-odoo ok, this should be good now if I didn't forget anything :)